modules/oadp-release-notes-1-1-2.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+:_content-type: REFERENCE
```

Comment:
I don't know if this will make any part of the toolchain complain but just in case :)
```suggestion
// * backup_and_restore/oadp-release-notes.adoc

:_content-type: REFERENCE
```

Reply:
Done